images/icons/template/mapping.json

```diff
@@ -51,5 +51,7 @@
   "repo-fetch": 61744,
   "repo-pull": 61745,
   "repo-push": 61746,
-  "provider-jira": 61747
+  "provider-jira": 61747,
+  "play-button": 61748,
+  "rocket-filled": 61749
```

Comment:
I don't see this used anywhere

Reply:
I'll use it instead of $(rocket)

![Screenshot 2024-10-18 at 10 38 27](https://github.com/user-attachments/assets/22554f13-c745-41cb-a3f6-b5f73d8cc879)